content/influxdb/cloud/notebooks/clean_data.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Normalize data
+description:
+weight: 102
+influxdb/v2.0/tags:
+menu:
+  influxdb_2_0:
+    name: Normalize data
+    parent: Notebooks
+---
+{{% note %}}
+**Notebooks is currently an early access feature.**
+[Submit a request](https://w2.influxdata.com/notebooks-early-access/ ) to be added to the queue, and we will send you a confirmation when you’ve been added to early access.
+{{% /note %}}
+
+Make it easier to compare data from different places by standardizing or normalizing it so that all measurements are between 0 and 1. This example walks through creating a notebook that normalizes response time using the [`map()`](/influxdb/cloud/reference/flux/stdlib/built-in/transformations/map/) function and writing it to a new bucket.
+
+## Requirements
+- This example uses [InfluxDB Cloud demo data](//influxdb/cloud/reference/sample-data/#influxdb-cloud-demo-data).
+- Create a destination bucket to write normalized data to. For details, see [Create a bucket](/influxdb/cloud/organizations/buckets/create-bucket/).
+
+
+## Normalize data with a notebook
+1. Create a new notebook (see [Create a notebook](/influxdb/v2.0/notebooks/create-notebook/).
+2. Add a **Metric Selector** cell to select a bucket to query data from:
+  - In the **Choose a bucket** dropdown, select **Website Monitoring Bucket**.
+  - Select the **response_time** field.
+  - Select the **mean** function from the aggregate selector in the upper-right.
+3. Add a **Flux Transformation** cell with the following script to bring in data from the previous cell and normalize it:
+  ```sh
+  __PREVIOUS_RESULT__
+    |> map(fn: (r) => ({ r with _value: r._value * r._value}))
+  ```
```

Comment:
The current response times are returned in seconds (fractions/float values). I more useful format may be response times in milliseconds:

~~~
```js
__PREVIOUS_RESULT__
  |> map(fn: (r) => ({ r with _value: r._value * 10000.0 }))
```
~~~

Reply:
I'm not sure this counts as "normalizing data"...@mnorriswalker what do you think?

Reply:
We could use the NOAA Water Sampling data...
Normalize `h2o_quality`
Convert units on `h2o_feet`

Reply:
If we're switching to the NOAA data, you wanted to normalize the `h2o_temperature` field by adjusting values to be similar to the `water_level` field, you would:

```
|> map(fn: (r) => ({ r with _value: float(v: r._value) / 10.0 }))
```

But there are some other things to work out before this is useful. You currently can't select data from multiple measurements using the metric selector cell. You can only do it with raw Flux.